docs/design/processing_trace_data/records.md

```diff
@@ -1,11 +1,20 @@
-# Record Object
+# Records Object
 
-In CARET, trace data is stored in tables, and latency calculation is performed by processing the tables.
-In addition to general table join processing, CARET defines classes which has originally defined join processing for latency calculation.
+CARET stores trace data as a table as shown below.
+
+| callback_start_timestamp | callback_end_timestamp |
+| ------------------------ | ---------------------- |
+| 0                        | 0.1                    |
+| 1                        | 1.1                    |
+| 2                        | 2.1                    |
+| ...                      | ...                    |
 
-This sections describes Record object which stores trace data and latency is.
+This table is processed to calculate latency, period, etc (See [Records Service](./records_service.md)).
```

Comment:
```suggestion
This table is referred to calculate latency, period, and etc. (See [Records Service](./records_service.md)).
```

Reply:
@hsgwa  
I think that latency is calculated from this table. If this is true, `refer` might be more appropriate word.